### ISSUE TITLE
Bug on migration tool when creating fields with type "double"

### ISSUE THREAD

**Issue by awesomeguy3** [yiisoft/yii2#12105]
Using php 7.0 and MariaDB. 

When creating a migration that contains fields of type "double", I normally used the function $this->double() which takes one parameter. That one parameter is precision, but when I entered any number or a string, or array of numbers (length and precision), it never set neither length nor precision in my DBMS. 

In MariaDB, I'm defining some field like DOUBLE(10, 2) where first number represents the length, and second number represents number of decimals, but I cannot define it in Yii2 migration with $this->double() function.

**Comment by cebe** [comment]
documentation reference: https://mariadb.com/kb/en/mariadb/double/

**Comment by awesomeguy3** [comment]
Exactly. When I went deeper in the code, I found out that the parameter $precision is marked only as integer (in phpdoc), and it's later used as length (not as precision). 

Example: 
if I put 
    $this->double(2); 
It will produce DOUBLE(2), 

But in MariaDB I need 2 parameters.

**Comment by sheillendra** [comment]
float too
